src/components/models/edit-measurement.js

```diff
@@ -49,59 +56,100 @@ const EditMeasurement = props => {
     }
   `)
 
-  const { units, breasts } = props.app.models[props.model]
-
   const updateMeasurement = evt => {
     let value = evt.target.value
     setValue(evt.target.value)
-    let mm = measurementAsMm(value, units)
-    let valid = typeof measurementAsMm(value, units) === 'number' ? true : false
+    let mm = measurementAsMm(value, currentModel.units)
+    let valid = typeof measurementAsMm(value, currentModel.units) === 'number' ? true : false
     if (valid) {
       setMm(mm)
       setUpdated(true)
     }
     setValid(valid)
   }
+
   const saveMeasurement = () => {
     const data = {
       measurements: {}
     }
-    data.measurements[props.measurement] = mm
+    data.measurements[currentMeasurement] = mm
     props.app.backend.saveModel(
       props.model,
       data,
-      props.app.frontend.intl.formatMessage({ id: 'measurements.' + props.measurement }),
+      props.app.frontend.intl.formatMessage({ id: 'measurements.' + currentMeasurement }),
       '/models/' + props.model
     )
   }
 
   let docs = null
   for (let node of Object.values(mdx.allMdx.edges)) {
     let m = node.node.parent.relativeDirectory.split('/').pop()
-    if (m === props.measurement.toLowerCase()) docs = node.node.body
+    if (m === currentMeasurement.toLowerCase()) docs = node.node.body
+  }
+  const label = props.app.frontend.intl.formatMessage({ id: 'measurements.' + currentMeasurement })
+
+  let measurementEstimate = false
+  let measurementInRange = value > 0
+  let helperText = () => null
+
+  if (currentModel.measurements.neckCircumference) {
+    measurementEstimate =
+      neckstimate(
+        currentModel.measurements.neckCircumference,
+        currentMeasurement,
+        currentModel.breasts
+      ) / 10
+    measurementInRange =
+      measurementDiffers(
+        currentModel.measurements.neckCircumference,
+        currentMeasurement,
+        value * 10,
+        currentModel.breasts
+      ) <= 2
+
+    // Only show measurementEstimate for non measurements.neckCircumference
+    helperText = () => {
+      if (currentMeasurement != 'measurements.neckCircumference') {
+        return (
+          <FormattedMessage
+            id="app.weEstimate"
```

Comment:
@joostdecock Is this sufficient or do I need to add it anywhere else?

Reply:
I think that's fine, though we might throw in a link there to documentation on how we do this (which remains to be written)